src/pages/document/[id].tsx

```diff
@@ -384,31 +381,42 @@ const FamilyPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({
               </div>
             </section>
 
-            {rootLevelConcepts.length > 0 && (
+            {Object.entries(rootLevelConcepts).length > 0 && (
               <section className="mt-8">
                 <Heading level={4}>Concepts</Heading>
-                <div className="flex text-sm">
-                  <ul className="flex flex-wrap gap-2">
-                    {rootLevelConcepts.map(([conceptName, count]) => {
-                      return (
-                        <li key={conceptName}>
-                          <ExternalLink
-                            className="capitalize"
-                            url={
-                              conceptName !== "Other"
-                                ? ROOT_LEVEL_CONCEPT_LINKS[conceptName]
-                                : "https://climatepolicyradar.wikibase.cloud/wiki/Main_Page"
-                            }
-                          >
-                            <Button color="clear" data-cy="view-family-concept" extraClasses="flex items-center text-sm">
-                              {conceptName} ({count})
-                            </Button>
-                          </ExternalLink>
-                        </li>
-                      );
-                    })}
-                  </ul>
-                </div>
+                {Object.entries(rootLevelConcepts).map(([rootConcept, subconcepts]) => (
+                  <div key={rootConcept} className="mb-4">
+                    {rootConcept !== "Other" && (
+                      <ExternalLink className="capitalize" url={ROOT_LEVEL_CONCEPT_LINKS[rootConcept]}>
+                        <Heading level={5} extraClasses="mb-2 capitalize">
+                          {rootConcept}
+                        </Heading>
+                      </ExternalLink>
+                    )}
+                    {rootConcept === "Other" && (
+                      <Heading level={5} extraClasses="mb-2 capitalize">
+                        {rootConcept}
+                      </Heading>
+                    )}
+
+                    <div className="flex text-sm">
+                      <ul className="flex flex-wrap gap-2">
+                        {Object.entries(subconcepts).map(([subconcept, { name, count, wikibaseId }]) => (
+                          <li key={subconcept}>
+                            <ExternalLink
+                              className="capitalize"
+                              url={`https://climatepolicyradar.wikibase.cloud/wiki/Item:${wikibaseId}#${encodeURIComponent(name)}`}
+                            >
+                              <Button color="clear" data-cy="view-family-concept" extraClasses="flex items-center text-sm">
```

Comment:
```suggestion
                              <Button color="clear" extraClasses="flex items-center text-sm">
```

as I don't think we use it anywhere and we're moving away from Cypress.